nomos-services/membership/src/lib.rs

```diff
@@ -201,12 +201,13 @@ where
                 let providers = self.backend.get_latest_providers(service_type).await;
 
                 if let Ok(providers) = providers {
-                    if tx.send(providers).is_err() {
+                    if !providers.1.is_empty() && tx.send(providers).is_err() {
```

Comment:
nitpick: Can we unpack providers instead?

```rust
if let Ok((_, providers)) = providers {
...
```

Reply:
`tx.send(providers).is_err()` This is sending the actual tuple with block numberr, that is why it is present.